content/en/api/payments.md

```diff
@@ -5,4 +5,99 @@ position: 41
 category: API
 ---
 
-Payments
\ No newline at end of file
+## Capture
+
+The capture endpoint allows capturing funds from a Single Payment authorisation, a Frequent Payment or a Frequent Payment authorisation. It also allows getting the details of a previous capture.
+
+### Capture payment
+
+`POST` `/capture/<payment id>`
+
+#### Parameters
+
+<div class="ep-protocol-list">
+
+  - `transaction_key`: `string` (<= 50 characters) with the internal key used to identify this transaction in the merchant's database.
+  - `capture_date`: `string` in the format `'Y-m-d'` with the capture date.
+  - `account`: *optional* `Object` for multi-account clients.
+    - `id`: `string` with the uuid of the account.
```

Comment:
You meant `uid`?

Reply:
No, I meant a https://en.wikipedia.org/wiki/Universally_unique_identifier . Again, nothing new, already the terminology of our documentation.

Reply:
Doesn't mean the previous documentation is correct. But I know the meaning now.